app/ui/view_nbgl.c

```diff
@@ -292,7 +278,9 @@ static void settings_screen_callback(uint8_t index, nbgl_content_t *content) {
 #ifdef APP_BLINDSIGN_MODE_ENABLED
     switches[BLINDSIGN_MODE].initState = app_mode_blindsign();
     switches[BLINDSIGN_MODE].text = "Blind sign";
-    switches[BLINDSIGN_MODE].subText = "";
+    if ((switches[BLINDSIGN_MODE].subText) == NULL) {
+        switches[BLINDSIGN_MODE].subText = "Allows signing transactions without seeing all fields";
```

Comment:
Is there enough space to display this on flex/stax? instead of "seeing", maybe "reviewing"? Otherwise, let's ask AI what it thinks about this description haha.